src/Microsoft.Framework.PackageManager/Restore/RestoreCommand.cs

```diff
@@ -93,10 +97,30 @@ public RestoreCommand(FrameworkName fallbackFramework)
 
             foreach (var category in ErrorMessages)
             {
-                Reports.Error.WriteLine("Errors in {0}".Red().Bold(), category.Key);
+                Reports.Error.WriteLine($"{Environment.NewLine}Errors in {category.Key}".Red().Bold());
                 foreach (var message in category.Value)
                 {
-                    Reports.Error.WriteLine("    {0}", message);
+                    Reports.Error.WriteLine($"    {message}");
+                }
+            }
+
+            if (Settings is Settings)
```

Comment:
You get MAAAAAAAAAAD props for writing this! Not as cool as intern Sonja's code, but still pretty cool.